_includes/header-nng.html

```diff
@@ -1,20 +1,71 @@
 <header class="nn-header">
-        <div class="nn-header-inner">
-                <a href="https://nng.nanomsg.org" class="nn-logo">
-                        <span class="nn-black">NN</span><span class="nn-tan">G</span>
-                </a>
-
-                <ul class="nn-nav-menu">
-                        <li class="nn-nav-item">
-                                <a href="https://nng.nanomsg.org/man/" class="nn-nav-link">Documentation</a>
-                        </li>
-                        <li class="nn-nav-item">
-                                <a href="https://discord.gg/Xnac6b9" class="nn-nav-link">Discord</a>
-                        </li>
-
-                        <li class="nn-nav-item">
-                                <a href="https://github.com/nanomsg/nng" class="nn-nav-link">GitHub</a>
-                        </li>
-                </ul>
+        <div class="nn-header-inner container">
+                <nav class="navbar" role="navigation" aria-label="navigation">
+                        <div class="navbar-brand">
+                                <a class="navbar-item" href="/">
+                                        <img width="54" height="36" src="/assets/image/NNg-white.png">
+                                </a>
+                                <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+                                        data-target="navbarBasicExample">
+                                        <span aria-hidden="true"></span>
+                                        <span aria-hidden="true"></span>
+                                        <span aria-hidden="true"></span>
+                                </a>
+                        </div>
+                        <div id="navbarBasicExample" class="navbar-menu">
+                                <div class="navbar-start"></div>
+                                <div class="navbar-end">
+                                        <div class="navbar-item">
+                                                <a href="/man/">Documentation</a>
+                                        </div>
+                                        <div class="navbar-item">
+                                                <a href="https://discord.gg/Xnac6b9">Discord</a>
```

Comment:
Given you have a GitHub icon, maybe also a Discord icon (and maybe a Docs icon of some kind?)